tests/unit/test_pheno_utils.py

```diff
@@ -338,32 +338,35 @@ def test_missing_ids_in_columns(test_data, columns, expected_indices):
         ("20Y6M", 20.5, "nb:FromISO8601"),
         ("P20Y6M", 20.5, "nb:FromISO8601"),
         ("20Y9M", 20.75, "nb:FromISO8601"),
+        ("20-25", 22.5, "nb:FromRange"),
+        ("20.00-25.00", 22.5, "nb:FromRange"),
     ],
 )
-def test_age_gets_converted(raw_age, expected_age, heuristic):
-    assert expected_age == pheno_utils.transform_age(raw_age, heuristic)
+def test_age_gets_converted(raw_age, expected_age, value_format):
+    assert expected_age == pheno_utils.transform_age(raw_age, value_format)
 
 
 @pytest.mark.parametrize(
-    "raw_age, incorrect_heuristic",
+    "raw_age, incorrect_format",
     [
         ("11,0", "nb:FromFloat"),
         ("11.0", "nb:FromISO8601"),
         ("20-30", "nb:FromBounded"),
+        ("20+", "nb:FromRange"),
```

Comment:
I think we'd want a test with "20" and "nb:fromRange" here to make sure the error gets thrown

Reply:
Or, I guess, "30-"? Would that lead to "15" as the answer?